priorityq/priorityq_test.go

```diff
@@ -0,0 +1,61 @@
+package priorityq
+
+import (
+	"container/heap"
+	"testing"
+)
+
+func TestPriorityQ(t *testing.T) {
+	// Some items and their priorities.
+	items := map[interface{}]int{
```

Comment:
Better be map[string]int.
But ok, let it be interface{}

Reply:
Fixed